src/features/vote/useTableValues.ts

```diff
@@ -48,7 +51,29 @@ export default function useTableValues(
         };
       });
 
-      setTableValues(tv);
+      const descriptionsAdded = Promise.all(
+        tv.map(async (el) => {
+          const isUmip = el.identifier.includes("Admin");
+          const umipNumber = isUmip
+            ? parseInt(el.identifier.split(" ")[1])
+            : undefined;
+
+          let description = "Price request.";
+
+          if (umipNumber) {
+            description = await (await fetchUmip(umipNumber)).description;
```

Comment:
if any of these fail, all of them will. might be ok, but consider handling the error somewhere if that is a problem

Reply:
Yeah I want to test this more with the CMS -- should probably be done on Kovan if we can figure out how to organize the data.